Description: man page cosmetic
 This patch fixes some little problems in two upstream manual pages.
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Forwarded: no

--- slurm-llnl-2.4.5.orig/doc/man/man1/srun.1
+++ slurm-llnl-2.4.5/doc/man/man1/srun.1
@@ -1777,7 +1777,9 @@ Total number of nodes in the job's resou
 Sets of node name, communication address and hostname for nodes allocated to
 the job from the cloud. Each element in the set if colon separated and each
 set is comma separated. For example:
-SLURM_NODE_ALIASES=ec0:1.2.3.4:foo,ec1:1.2.3.5:bar
+.na
+SLURM_NODE_ALIASES\:=\:ec0:1.2.3.4:foo,ec1:1.2.3.5:bar
+.ad
 .TP
 \fBSLURM_NODEID\fR
 The relative node ID of the current node
--- slurm-llnl-2.4.5.orig/doc/man/man5/slurm.conf.5
+++ slurm-llnl-2.4.5/doc/man/man5/slurm.conf.5
@@ -1,4 +1,4 @@
-.TH "slurm.conf" "6" "April 2012" "slurm.conf 2.4" "Slurm configuration file"
+.TH "slurm.conf" "5" "April 2012" "slurm.conf 2.4" "Slurm configuration file"
 
 .SH "NAME"
 slurm.conf \- Slurm configuration file
--- slurm-llnl-2.4.5.orig/doc/man/man8/spank.8
+++ slurm-llnl-2.4.5/doc/man/man8/spank.8
@@ -357,7 +357,7 @@ This function returns \fBESPANK_SUCCESS\
 struct spank_option \fIopt\fR has been used by the user. If \fIoptargp\fR
 is non-NULL then it is set to any option argument passed (if the option
 takes an argument). The use of this method is \fIrequired\fR to process
-options in \fBjob_script\fR context (\fBslurm_spank_job_prolog\R and
+options in \fBjob_script\fR context (\fBslurm_spank_job_prolog\fR and
 \fBslurm_spank_job_epilog\fR).
 
 .SH "CONFIGURATION"
